.cirrus.yml

```diff
@@ -53,8 +53,8 @@ build_task:
   cleanup_before_cache_script: cleanup_maven_repository
 
 mend_scan_task:
-  only_if: >-
-    $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_TAG == "" && ($CIRRUS_BRANCH == "master" || $CIRRUS_BRANCH =~ "branch-.*")
+  <<: *ONLY_SONARSOURCE_QA
```

Comment:
This changed the condition a little bit and the mend task is running on PR now.

Reply:
Correct, I will revert the change.